kafka-connect-s3/docs/s3_connector.rst

```diff
@@ -314,3 +314,87 @@ schema generator class to its default value.
 Finally, schema evolution is disabled in this example by setting ``schema.compatibility`` to ``NONE``, as explained above.
 
 
+Write raw message values into S3
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is possible to use the S3 connector to write out the unmodified original message values into
+newline-separated files in S3. We accomplish this by telling Connect to not deserialize any of the
```

Comment:
I am wondering whether message itself can contain escaped newline character?

Reply:
I agree that's worth bringing up here. [The docs for the FORMAT_BYTEARRAY_LINE_SEPARATOR_CONFIG do specify](https://github.com/confluentinc/kafka-connect-storage-cloud/blob/master/kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java#L318-L319):

> An input record that contains the line separator will look like multiple records in the output S3 object.

So, the connector won't break, but the output may not be interpreted the way you expect if it contains the separator, and users need to be aware of that responsibility.

Reply:
It's probably appropriate to discuss this detail later in the example, when we mention `format.bytearray.separator`.